indexer/pkg/config/config.go

```diff
@@ -0,0 +1,77 @@
+package config
+
+import (
```

Comment:
I am not sure why this pkg is required. We should use urfave/cli to write the options. You can set defaults and also specify the ENV variables which map to the flags. We use this in all the other go services. All this custom handling is not required in my opinion.

https://github.com/primev/mev-commit/blob/main/oracle/cmd/main.go

This ensures just one file which can create a custom config for the node to start.